pkg/controller/pagerdutyintegration/pagerdutyintegration_controller_test.go

```diff
@@ -167,13 +168,15 @@ func testPagerDutyIntegration() *pagerdutyv1alpha1.PagerDutyIntegration {
 }
 
 // testClusterDeployment returns a fake ClusterDeployment for an installed cluster to use in testing.
-func testClusterDeployment(isInstalled bool, isManaged bool, hasFinalizer bool, isDeleting bool) *hivev1.ClusterDeployment {
+func testClusterDeployment(isInstalled bool, isManaged bool, hasFinalizer bool, isDeleting bool, isFake bool) *hivev1.ClusterDeployment {
```

Comment:
Can we add a test for a "fake" ClusterDeployment?

Reply:
I wrote a test for it on line 247:
https://github.com/openshift/pagerduty-operator/pull/152/files#diff-5651ca632758a6747842fd344dc5effcc1cdff0b5892c58727ba750e49752a62R247